BaseClasses.py

```diff
@@ -901,24 +901,9 @@ def remove(self, item: Item):
             self.stale[item.player] = True
 
 
-@unique
-class RegionType(IntEnum):
-    Generic = 0
-    LightWorld = 1
-    DarkWorld = 2
-    Cave = 3  # Also includes Houses
-    Dungeon = 4
-
-    @property
-    def is_indoors(self) -> bool:
-        """Shorthand for checking if Cave or Dungeon"""
-        return self in (RegionType.Cave, RegionType.Dungeon)
-
-
 class Region:
     name: str
-    type: RegionType
-    hint_text: str
+    hint: Optional[str]
```

Comment:
can we rename this to _hint_text, so it matches what we have for Location, and have it open to the getattr treatment we have in Location? I'm not sure if that would break anything.

Reply:
i think lttp will be fine as that's what i wrote the property for but this will absolutely break OoT